InventoryCatalog/Setup/Patch/Schema/InitializeDefaultStock.php

```diff
@@ -53,7 +71,9 @@ public function __construct(
     public function apply()
     {
         $this->createDefaultSource->execute();
+        $this->createDefaultSourceTypes->execute();
```

Comment:
Hmm. I afraid it should work for pure MSI installation but could fail in the existing MSI installation upgrade.
@sidolov, what do you think?

Reply:
I gonna make a new patch to create types also assign.